### ISSUE TITLE
Uncaught TypeError: Cannot read property 'scrub' of undefined

### ISSUE THREAD

**Issue by Liingon** [jonkwheeler/ScrollScene#3]
I tried to use the ScrollObserver to control a video, but got the following error:
```
Uncaught TypeError: Cannot read property 'scrub' of undefined
    at ScrollObserver.js:153
    at Array.forEach (<anonymous>)
    at IntersectionObserver.observerCallback (ScrollObserver.js:149)
```

This is what I used to initilize:
```
        new ScrollObserver({
            triggerElement: trigger,
            video: {
                element: el
            },
        });
```

It seems to be this line that is the problem, I got around it by including an empty gsap timeline.
https://github.com/jonkwheeler/ScrollScene/blob/f0d63b7709ef3ca12b50d02ef24af3ced14e71c1/src/ScrollObserver.ts#L334

**Comment by jonkwheeler** [comment]
@Liingon is `el` a `<video>`?

**Comment by Liingon** [comment]
Yep!

**Comment by jonkwheeler** [comment]
Let me take a look. I just added that feature and forgot to test video again.

**Comment by jonkwheeler** [comment]
@Liingon I just released a new version 0.0.6. I needed extra parentheses. I also added a video story. I need to fix the video up a bit, but it's working for now. It will play once the triggerElement is visible on the page, and pause after it's not. https://scrollscene.jonkwheeler.now.sh/?path=/story/scrollobserver-video--basic-example
